src/main/java/org/kainos/ea/cli/Role.java

```diff
@@ -1,7 +1,7 @@
 package org.kainos.ea.cli;
 
 public enum Role {
-    EMPLOYEE(0),
+    EMPLOYEE(2),
```

Comment:
Is the role a table in the database or just a int? If there is a role table for admin/user it would be better for you to get the role of the database and assign it based on that, else this code could break really easily